.vscode/settings.json

```diff
@@ -1,6 +1,5 @@
 {
     "files.associations": {
         "*.xsh": "python"
-    },
-    "python.formatting.provider": "yapf"
```

Comment:
I deleted this entirely since this setting is no longer used by VScode:
https://code.visualstudio.com/docs/python/formatting